### PR TITLE
Fix for Xbox One controller right Y axis on macOS

### PR DESCRIPTION
**Pull request by willco007**

### CODE REVIEW

File: gamecontrollerdb.txt [hunk 1]
```diff
@@ -875,7 +875,7 @@ xinput,XInput Controller,a:b0,b:b1,back:b6,dpdown:h0.4,dpleft:h0.8,dpright:h0.2,
...
 030000005e040000ea02000000000000,Xbox Wireless Controller,a:b0,b:b1,back:b9,dpdown:b12,dpleft:b13,dpright:b14,dpup:b11,guide:b10,leftshoulder:b4,leftstick:b6,lefttrigger:a2,leftx:a0,lefty:a1,rightshoulder:b5,rightstick:b7,righttrigger:a5,rightx:a3,righty:a4,start:b8,x:b2,y:b3,platform:Mac OS X,
-030000005e040000fd02000003090000,Xbox Wireless Controller,a:b0,b:b1,back:b16,dpdown:h0.4,dpleft:h0.8,dpright:h0.2,dpup:h0.1,guide:b15,leftshoulder:b6,leftstick:b13,lefttrigger:a5,leftx:a0,lefty:a1,rightshoulder:b7,rightstick:b14,righttrigger:a4,rightx:a2,righty:a3,start:b11,x:b3,y:b4,platform:Mac OS X,
+030000005e040000fd02000003090000,Xbox Wireless Controller,a:b0,b:b1,back:b16,dpdown:h0.4,dpleft:h0.8,dpright:h0.2,dpup:h0.1,guide:b15,leftshoulder:b6,leftstick:b13,lefttrigger:a5,leftx:a0,lefty:a1,rightshoulder:b7,rightstick:b14,righttrigger:a4,rightx:a2,righty:a5,start:b11,x:b3,y:b4,platform:Mac OS X,
```
Comment (cgutman) [inline gamecontrollerdb.txt:878]: This doesn't look right. `a5` is already `lefttrigger`.

**Comment by willco007** [comment]
Let me figure out what left trigger is coming back from HID as...

**Comment by willco007** [comment]
I'm going to close this for now; I will re-open if I need to after sorting out a bit of stuff on my end. Sorry for the noise.

**Comment by offalynne** [comment]
No problem, and thank you for checking ! Looks like this is the Xbox One S controller over BT: https://devicehunt.com/view/type/usb/vendor/045E/device/02FD. If there is an issue with the mapping I suspect it will need to be taken up with the SDL project proper @ https://github.com/libsdl-org/SDL since first-party device support is a priority. I'll have a look myself and see if I can repro anything interesting, feel free to add details to this thread in the future if you have any.

**Pull request closed without merging**